spark/src/main/java/org/opensearch/sql/spark/config/SparkExecutionEngineConfig.java

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.sql.spark.config;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.google.gson.Gson;
+import lombok.Data;
+
+@Data
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class SparkExecutionEngineConfig {
```

Comment:
does the name represents the actual execution config or only the client job dispatch ?

Reply:
This carries spark configuration set by user.